invenio_rdm_records/services/schemas/record.py

```diff
@@ -66,7 +66,7 @@ class Meta:
     versions = NestedAttribute(VersionsSchema, dump_only=True)
     parent = NestedAttribute(RDMParentSchema)
     is_published = fields.Boolean(dump_only=True)
-    is_deleted = fields.Boolean(dump_only=True)
+    has_draft = fields.Boolean(dump_only=True)
```

Comment:
for reviewers: I need this exposed, do you see any issues ?

is_deleted was not correct attribute, the deletion status is coming from system field